Charts/grist/README.md

```
@@ -0,0 +1,157 @@
+| `config.APP_TITLE` | Custom app title | `"Grist"` |
+| ...
+
+## Usage
+
+1. Modify the values.yaml file to match your requirements
+2. Install the chart
+3. Access Grist through the configured ingress or service
+
+For more information, visit the [Grist documentation](https://support.getgrist.com/self-managed/).
+
+## Configuration
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)